### ISSUE TITLE
Incomplete example in document

### ISSUE THREAD

**Issue by sebastus** [Azure/azure-content#5035]
https://azure.microsoft.com/en-us/documentation/articles/best-practices-resource-manager-state/

in this document, at the very bottom is an example of consuming output from an embedded template.  

In what context do the lines creating "masterIpAddress" go?  Is it part of resource?  somewhere else?  I think we need an example of exactly how output is consumed in the article.

**Comment by BernieWhite** [comment]
Agreed the example is a little unclear. The "World Class ARM Templates Considerations and Proven Practices" paper linked on that article has an example in a resource on page 12.
